AWESOME_RGF.md

```diff
@@ -0,0 +1,26 @@
+Awesome RGF
+===========
+This page contains a curated list of blogs, competitions solutions about RGF.
+It is inspired by [awesome-xgboost](https://github.com/dmlc/xgboost/blob/master/demo/README.md),
+
+Please feel free to send a pull request if you find things that belongs to here.
+
+Contents
```

Comment:
I think that at present this doc is rather small, and `Contents` will just occupy place without any useful mission.